packages/core/src/components/spinner/_spinner.scss

```diff
@@ -105,6 +105,11 @@ Styleguide components.progress.spinner.js.svg
   }
 }
 
+@keyframes pt-spinner-animation {
+  from { transform: rotate(0deg); }
+  to   { transform: rotate(360deg); }
+}
+
 $spinner-width: $pt-grid-size * 10 !default;
 $spinner-width-factor: 0.5 !default;
 $spinner-width-factor-small: 0.24 !default;
```

Comment:
We want the spinner thickness to be the same, no matter what size it is. SVG view box makes it tricky to deal with :( Definitely something we should improve